lib/index.js

```diff
@@ -16,30 +16,37 @@ const File = require('vinyl')
 //               resulting module exports an object with views keyed by short
 //               file paths. If not provided, each module exports its view as a
 //               string.
+//   .prefix  -- prepend a prefix (e.g. file path) to the filename for each of
+//               the HTML files supplied. Do not include the trailing slash, as
+//               it gets added automatically.
+//   .global  -- override the variable to which the list of HTML strings is
+//               assigned to. For example `window.templates`. If not provided,
+//               `module.exports` will be used for compatibility with node.js
 module.exports = options => (
   options && options.concat
-  ? concatMode(options.concat)
+  ? concatMode(options)
   : baseMode()
 )
 
-function concatMode (concat) {
-  if (typeof concat !== 'string') {
-    throw Error(`Option 'concat' must be a string, got: ${concat}`)
+function concatMode (options) {
+  if (typeof options.concat !== 'string') {
+    throw Error(`Option 'concat' must be a string, got: ${options.concat}`)
   }
 
+  options.global = options.global || 'module.exports'
```

Comment:
Default should remain as `module.exports` for non-breaking changes. Most of the rest of this function is to manage the fact that we're now passing the full `options` object, as opposed to just the `options.concat` property.